mappings/net/minecraft/world/gen/feature/GlowLichenFeature.mapping

```diff
@@ -9,7 +9,7 @@ CLASS net/minecraft/class_5779 net/minecraft/world/gen/feature/GlowLichenFeature
 	METHOD method_33392 shuffleDirections (Lnet/minecraft/class_5780;Ljava/util/Random;)Ljava/util/List;
 		ARG 0 config
 		ARG 1 random
-	METHOD method_33393 excludeDirection (Lnet/minecraft/class_5780;Ljava/util/Random;Lnet/minecraft/class_2350;)Ljava/util/List;
+	METHOD method_33393 shuffleDirections (Lnet/minecraft/class_5780;Ljava/util/Random;Lnet/minecraft/class_2350;)Ljava/util/List;
```

Comment:
Is this name right?

Reply:
Yes, the method shuffles the directions
it is identical to the other one named that, except it excludes the direction parsed as exclude